### PR TITLE
docs(@icons): Change icons resolution to 16px

### PR DESCRIPTION
**Pull request by ilzamcmed**
Jira issue: https://liferay.atlassian.net/browse/LPS-201961

### CODE REVIEW

**Comment by ilzamcmed** [comment]
Hey @pat270 , this PR is ready for review
Thanks!

File: packages/clay-css/src/images/icons/Union 5.svg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!--
+* SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2022 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
+<svg viewBox="0 0 16 16" xmlns="http://www.w3.org/2000/svg"><path class="lexicon-icon-outline" d="m11.647.365 4.037 4.031c.875 1.019-.23 2.375-1.353 1.785l-.987 1.975-5.625-5.438L9.83 1.674C9.25.64 10.541-.635 11.647.365Zm1.315 13.965L1.65 3.019c-1.275-1.3.506-3.144 1.944-1.881L14.906 12.45c1.312 1.457-.469 3.238-1.944 1.882ZM0 16l4.375-6.766L6.8 11.652 0 16Z" fill="#000"/></svg>
```
Comment (ilzamcmed) [inline packages/clay-css/src/images/icons/Union 5.svg:7]: Waiting confirmation on how this icon should be called

File: packages/clay-css/src/images/icons/Vector 1.svg [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<!--
+* SPDX-FileCopyrightText: © 2022 Liferay, Inc. <https://liferay.com>
+* SPDX-FileCopyrightText: © 2022 Contributors to the project Clay <https://github.com/liferay/clay/graphs/contributors>
+*
+* SPDX-License-Identifier: BSD-3-Clause
+-->
```
Comment (ilzamcmed) [inline packages/clay-css/src/images/icons/Vector 1.svg:6]: Waiting confirmation on how this icon should be called

**Comment by pat270** [comment]
@ilzamcmed it looks pretty good! The only change we need is to remove `fill="#000"` from many of the icons. It doesn't inherit color changes.

Screen Shot 2023-12-14 at 2 44 42 PM

All the icons should be blue not black.

**Review by pat270 — CHANGES_REQUESTED** [review]
Need to remove `fill="#000"`

**Comment by ilzamcmed** [comment]
Thanks @pat270 
I believe everything is ok now. The only question I have is related to the icons called `Union` (didn't find any related one) and `Vector` (this one is similar to the geolocation icon but a bit small).

**Review by pat270 — CHANGES_REQUESTED** [review]
@emiliano-cicero There were some icons not included in the zip. These icons are still `viewBox="0 0 512 512"`. Are we deleting them? If we do, there will be complications and we will probably have to provide backward compatibility.

```
add-role.svg
align-right.svg
anonymize.svg
arrow-right-full.svg
arrow-up-full.svg
automatic-translate.svg
bell-full.svg
bolt.svg
change-list-disabled.svg
comments.svg
custom-field.svg
desktop.svg
devices.svg
diagonal-line.svg
embed.svg
environment-connected.svg
environment-disconnected.svg
environment.svg
geolocation.svg
import-export.svg
info-panel-closed.svg
lock-dots.svg
megaphone-full.svg
minus-circle.svg
order-arrow.svg
order-form-pencil.svg
order-list-down.svg
order-list-up.svg
pages-tree.svg
pin-full.svg
product-menu.svg
rectangle.svg
relationship.svg
remove-role.svg
reset.svg
share-alt.svg
sign-in.svg
social-twitter.svg
sticky.svg
textbox.svg
third-party.svg
twitter-x.svg
unpin.svg
upload-multiple.svg
urgent.svg
warning-full.svg
```

**Comment by pat270** [comment]
@ilzamcmed we can rename the two icons

```
union 5 => unpin.svg
vector 1 => geolocation.svg
```

**Comment by ilzamcmed** [comment]
> @emiliano-cicero There were some icons not included in the zip. These icons are still `viewBox="0 0 512 512"`. Are we deleting them? If we do, there will be complications and we will probably have to provide backward compatibility.
> 
> ```
> add-role.svg
> align-right.svg
> anonymize.svg
> arrow-right-full.svg
> arrow-up-full.svg
> automatic-translate.svg
> bell-full.svg
> bolt.svg
> change-list-disabled.svg
> comments.svg
> custom-field.svg
> desktop.svg
> devices.svg
> diagonal-line.svg
> embed.svg
> environment-connected.svg
> environment-disconnected.svg
> environment.svg
> geolocation.svg (vector 1.svg)
> import-export.svg
> info-panel-closed.svg
> lock-dots.svg
> megaphone-full.svg
> minus-circle.svg
> order-arrow.svg
> order-form-pencil.svg
> order-list-down.svg
> order-list-up.svg
> pages-tree.svg
> pin-full.svg
> product-menu.svg
> rectangle.svg
> relationship.svg
> remove-role.svg
> reset.svg
> share-alt.svg
> sign-in.svg
> social-twitter.svg
> sticky.svg
> textbox.svg
> third-party.svg
> twitter-x.svg
> unpin.svg (union 5.svg)
> upload-multiple.svg
> urgent.svg
> warning-full.svg
> ```

Should we consider the flags also? They are all `viewBox="0 0 512 512"`

**Comment by pat270** [comment]
Yes flags should be converted too.

**Comment by ilzamcmed** [comment]
Hey @emiliano-cicero , do you have any updates on the icons?
Thanks!

**Comment by ilzamcmed** [comment]
To be sent in another moment after design icons review

**Pull request closed without merging**